i18n/zh/docusaurus-plugin-content-docs/current/setup/install-with-keadm.md

```diff
@@ -15,6 +15,25 @@ KubeEdge 对 Kubernetes 的版本兼容性，更多详细信息您可以参考 [
 - 需要超级用户权限（或 root 权限）才能运行。
 - `keadm beta`功能在 v1.10.0 上线，如果您需要使用相关功能，请使用 v1.10.0 及以上版本的 keadm。
```

Comment:
It looks like `keadm beta` is not available. Could you please check it? If so, you can remove the `keadm beta` related section.

Reply:
![image](https://github.com/kubeedge/website/assets/13061869/1e920efc-3731-4dd5-87e5-cbc826703351)
It looks in keadm v1.17.0, the keadm beta command still in help pages.

Reply:
![image](https://github.com/kubeedge/website/assets/13061869/b6f86e38-a3a0-437c-975e-d24da34eb762)
It looks in keadm v1.17.0, the keadm beta still in help page

Reply:
At that time, `keadm beta join` seemed to be used to transition to deploying cloudcore in a containerized manner, but now `keadm join` has deployed cloudcore in a containerized manner by default. So what do you think about deleting the relevant content of keadm beta? @xuemingdi @Shelley-BaoYue

Reply:
@wbc6080  Thanks, I have removed `keadm beta join` from this document.